screenpipe-vision/build.rs

```diff
@@ -1,20 +1,48 @@
 #[cfg(target_os = "macos")]
-use std::env;
-
-#[cfg(target_os = "macos")]
-use std::path::PathBuf;
+use std::{env, process::Command, path::PathBuf};
```

Comment:
why both build.rs and pre_build.js? seems redundant

Reply:
also predict it breaks OCR link

Reply:
build.rs is for 'cargo build' so you don't need to compile binaries separately
pre_build.rs for 'bun tauri build'